packages/cli-kit/src/lib/FlagshipCodeLogger.ts

```diff
@@ -362,18 +362,6 @@ export class FlagshipCodeLogger {
         this.getMessage(messages, false),
       );
 
-      if ((message as any) instanceof Error) {
-        const errorStackArr = (message as any).stack.split(
-          '\n',
-        ) as unknown as Array<string>;
-
-        errorStackArr.shift();
-
-        FlagshipCodeManager.shared.emit(
-          'onLog',
-          chalk.dim('\n' + errorStackArr.join('\n')),
-        );
-      }
```

Comment:
this needs to be looked into further

Reply:
there is no great way to split the stack from the error message, so we may just want to display the stack instead since it includes both - the stack can be useful.